docs/v2-system-addon/data_events.md

```diff
@@ -264,3 +265,46 @@ perf: {
   "visibility_event_rcvd_ts": 2,
 }
 ```
+
+## Pocket pings
```

Comment:
Would be better to write Top Story pings or even more general Section/Card pings, but happy to fix it up later.